axiom/monitors.go

```diff
@@ -28,6 +28,13 @@ const (
 	AboveOrEqual // AboveOrEqual
 )
 
+type Type uint8
```

Comment:
Please add a comment to this type explaining what it is. See the Operator comment for reference.